cluster/common/flux/variables.tf

```diff
@@ -4,6 +4,18 @@ variable "flux_repo_url" {
   default = "https://github.com/weaveworks/flux.git"
 }
 
+# container registry to download flux image
+variable "flux_image_repository" {
+  type = "string"
+  default = "docker.io/weaveworks/flux"
```

Comment:
You have the default here, yet it needs to be passed in via aks-gitops?

Reply:
Yes. The scenario is flux module has the default value, but wanted to override the it only in `aks-gitops` implementation.